src/compute/flexbox.rs

```diff
@@ -143,6 +143,12 @@ struct AlgoConstants {
     /// Is the wrap direction inverted
     is_wrap_reverse: bool,
 
+    /// The item's min_size style
+    min_size: Size<Option<f32>>,
+    /// The item's max_size style
+    max_size: Size<Option<f32>>,
+    // /// The item's aspect_ratio style
+    // aspect_ratio: Option<f32>,
```

Comment:
Any specific plans for this commented code?

Reply:
No, that can go. It's copied from the `Style` as a cache, but I didn't end up using it.